src/lib.rs

```diff
@@ -126,6 +126,9 @@ pub use waker::Waker;
 #[cfg(unix)]
 pub mod unix {
```

Comment:
Maybe add a conditional in this mod and `mod net` as well, because without any feature these are now two empty modules, so might as well not have them.